app.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+
+var express = require('express');
+var fs = Promise.promisifyAll(require('fs'));
```

Comment:
Better to use a local Promise implementation here. The global es6 Promise doesn't actually have `promisifyAll`.

Example:
```javascript
var P = require('bluebird');
var fs = P.promisifyAll(require('fs'));
```